en/1-who-we-are.md

```diff
@@ -11,4 +11,4 @@ To build the WoM and Meeds, the association leverages the Ethereum platform and
 
 The association is structured as a Decentralized Autonomous Organization (DAO) where the governance token holders vote on all important organizational changes such as electing the board every year.
```

Comment:
```suggestion
The establishment is structured as a Decentralized Autonomous Organization (DAO) where the governance token holders vote on all important organizational changes such as electing the board every year.
```

Reply:
The word "establishment" will do, in my opinion, as the phrase "association" has not done a better job of describing what Meeds DAO is.

Reply:
well, Association is the legal status of Meeds https://en.wikipedia.org/wiki/Swiss_association.

But I'll find another way to avoid the word.

Reply:
chapter rewritten to limit use of the word "association" while still introducing Hubs and the WoM